Makefile

```diff
@@ -667,6 +667,8 @@ pylint:
 # sample lib for MXNet extension dynamically loading custom operator
 sample_lib:
 	$(CXX) -shared -fPIC -std=c++11 example/extensions/lib_custom_op/gemm_lib.cc -o libsample_lib.so -I include/mxnet
+subgraph_lib:
+	$(CXX) -shared -fPIC -std=c++11 example/extensions/lib_subgraph/subgraph_lib.cc -o libsubgraph_lib.so -I include/mxnet
```

Comment:
Will it be generated to the root folder of mxnet?

Reply:
Yes, for now same as custom op. @rondogency will be addressing this in the customOp #2 PR (coming this week)

Reply:
Thank you. I just thought we need a centralized place to put these so files. Scattering them in the root folder is not very neat for me~